<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Activity Logger</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0,0,0">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="labelDesc">
        <property name="text">
         <string>Description:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QTimeEdit" name="endTimeEdit"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelEnd">
        <property name="text">
         <string>End Time:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelDate">
        <property name="text">
         <string>Date:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QTimeEdit" name="startTimeEdit"/>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="addButton">
        <property name="text">
         <string>Add Activity</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelStart">
        <property name="text">
         <string>Start Time:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDateEdit" name="dateEdit">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
        <property name="date">
         <date>
          <year>2000</year>
          <month>1</month>
          <day>1</day>
         </date>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="descriptionEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="showAllButton">
      <property name="text">
       <string>Mostrar todas</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="deleteButton">
      <property name="text">
       <string>Eliminar seleccionada</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="countAllButton">
      <property name="text">
       <string>Total actividades</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="countByTypeButton">
      <property name="text">
       <string>Contar tipo</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="typeLineEdit">
      <property name="text">
       <string>&quot;Ej: Correr&quot;  </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="activityListWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>620</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
